resources/views/user/index.blade.php

```diff
@@ -38,6 +41,11 @@
                             @if(\LibreNMS\Authentication\LegacyAuth::getType() == 'mysql')
                             <td>{{ $user->enabled }}</td>
                             @endif
+                            @if(\LibreNMS\Config::get('twofactor'))
+                                @if(isset($twofactor[$user->user_id]))
+                                <td>{{ $twofactor[$user->user_id]->pref }}</td>
```

Comment:
I suggest you use `UserPref::getPref($user, 'twofactor')`, but I think the code should be updated to be more efficient (as below).  You should also pre-load the preferences relationship.  As in my other comment.

```php
    public static function getPref(User $user, $pref)
    {
        if ($user->relationLoaded('preferences')) {
            return optional($user->preferences->firstWhere('pref', $pref))->value;
        }

        return $user->preferences()->where('pref', $pref)->value('value');
    }
```